docs/spec/channel.md

```diff
@@ -391,6 +384,23 @@ Channels SHOULD expose a variety of metrics, including, but not limited to:
 Metrics SHOULD be enabled by default, with a configuration parameter included to
 disable them if desired.
 
+The Channel MUST recognize and pass through all tracing information from sender
+to subscribers using [W3C Tracecontext](https://w3c.github.io/trace-context/),
+although internally it MAY use another mechanism(s) to propagate the tracing
+information. The Channel SHOULD sample and write traces to the location
+specified in
+[`config-tracing`](https://github.com/knative/eventing/blob/master/config/config-tracing.yaml).
```

Comment:
Same applies for broker

Reply:
It seems to me the observabiliity parts of broker and channel should be exactly the same. I am wondering if it worths it is own spec and just reference the spec in both channel and broker. Adam also mentioned that whether the broker should have some common metrics, but I remembered the last conversation ended with not going to implement it.

Reply:
@zhongduo thats a fair point.
In the current spec, the channel has an additional requirement of passing through tracing info from sender to subscribers. It sounds like that would be applicable for the Broker as well ?

Reply:
> It sounds like that would be applicable for the Broker as well ?

Yep, and similar to channel, there is no constraint on how to propagate tracing informations

Reply:
I think it will be very confusing if there are difference between Channel and broker, as you can build a "broker" using channels.

Reply:
> as you can build a "broker" using channels.

But that would be an implementation detail of how you build that particular broker

Reply:
I wasn't really thinking about "how to build the broker". My point is that in tracing side, most requirements in broker will apply in channel as well, so just wondering how we can avoid such an inconsistency. I do understand theoretically they can be very different, but at least now they looks related to me.